extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -28,8 +28,8 @@ const ATTR_PREFIX = 'amp-x-';
 export class AmpExperiment extends AMP.BaseElement {
 
   /** @override */
-  isLayoutSupported(unusedLayout) {
-    return true;
+  isLayoutSupported(layout) {
+    return layout == Layout.NODISPLAY || layout == Layout.CONTAINER;
```

Comment:
Setting it to hidden in `buildCallback` is too late - it might have already been visible for a long time before the `buildCallback` had a chance to download and upgrade. Instead, @muxin explicitly set it to `display: none;` in another PR per my request. This is just a follow up to make sure users don't even try to set width/height and other layout attributes on this element.